README.md

```diff
@@ -109,10 +123,11 @@ STATICFILES_DIRS = (
 Restart the apache server to put all changes into effect. 
 
 ```
-/etc/init.d/apache2 reload
+sudo /etc/init.d/apache2 reload
 ```
 
-###Migrate the Database Schema using South
```

Comment:
Only if there are model changes not accounted for in the branch. I would prefer to rename the section and take out the "make migrations" part, but the previous folks had it written there. Up for debate ;)
Either way the next three lines are necessary to load the schema into the database, but that can be under a different section name